libs/evcs-service/src/main/java/uk/gov/di/ipv/core/library/client/EvcsClient.java

```diff
@@ -72,7 +77,10 @@ public EvcsGetUserVCsDto getUserVcs(
             var evcsHttpResponse = sendHttpRequest(httpRequestBuilder.build());
 
             EvcsGetUserVCsDto evcsGetUserVCs =
-                    OBJECT_MAPPER.readValue(evcsHttpResponse.body(), new TypeReference<>() {});
+                    evcsHttpResponse.statusCode() != 404
```

Comment:
Ah, so it returns a 404 with a different response structure if the user has no VCs in the store? That's a bit annoying 😞

Reply:
404 is returned as error. (seems like for scenario when user does not exist).